pkg/linear-pools/contracts/silo-linear-pool/SiloLinearPoolRebalancer.sol

```diff
@@ -63,7 +61,7 @@ contract SiloLinearPoolRebalancer is LinearPoolRebalancer {
     }
 
     function _getRequiredTokensToWrap(uint256 wrappedAmount) internal view override returns (uint256) {
-        uint256 rate = _exchangeRateModel.calculateExchangeValue(_shareToken);
+        uint256 rate = _calculateExchangeValue(_shareToken);
         return wrappedAmount.mulDown(rate) + 1;
```

Comment:
This seems correct but I'm not a fan of reusing this function in `_unwrapTokens()` because it creates this odd sequence:

1. Compute the rate.
2. Convert wrapped token amount.
3. Add one.
4. Subtract one.

I think it would be better to create a new function in the `SiloExchangeRateModel` that performs this conversion in steps 1 and 2. Then, use that function by itself within `_unwrapTokens()` and add one to it in `_getRequiredTokensToWrap()`. The [`EulerLinearPoolRebalancer`](https://github.com/orbcollective/linear-pools/blob/c79ec30158e1b0072d159183c8b62740a8ef81c3/pkg/linear-pools/contracts/euler-linear-pool/EulerLinearPoolRebalancer.sol) has a good example of this workflow.